.github/workflows/documentation.yml

```
@@ -0,0 +1,17 @@
+name: test documentation
+
+on: [ push, pull_request ]
+
+jobs:
+    tests:
+        name: documentation
```

```suggestion
        name: Documentation
```
The names are shown on GitHub, so we should make them pretty! ;o)